src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/DesignTimeTargets/Microsoft.CSharp.DesignTime.targets

```diff
@@ -64,6 +64,41 @@
 
   <!-- Targets -->
 
+  <!-- Razor cohosting support for projects that don't use the Razor SDK -->
+
+  <PropertyGroup>
+    <_HasRazorOrCshtmlFiles>false</_HasRazorOrCshtmlFiles>
+  </PropertyGroup>
+
+  <!-- .NET Framework uses a different editor in VS -->
+  <ItemGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework' And 'UsingMicrosoftNETSdkRazor' != 'true'">
+    <!-- Collect all Razor and CSHTML files -->
+    <_RazorOrCshtmlFiles Include="**\*.razor;**\*.cshtml" />
```

Comment:
Are these items already in an item group somewhere that we can piggy back off? It'd be nice to avoid adding more globs.

Reply:
I don't think there is any consistency here, they could be in Content or None, but they could be nowhere. My last contribution to this repo was actually around the same issue, because Maui and Bing did weird and different things: https://github.com/dotnet/project-system/pull/7417

This really is a last resort catch _all_ we're having to write.